### PR TITLE
Add additional admins for VAOS

### PR DESCRIPTION
**Pull request by jbalboni**
## Description of change
This adds additional users who should be able to control the feature flags for VAOS.

#### Unique to this PR
- [ ] Admins are correctly specified

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by jbalboni** [comment]
@johnpaulashenfelter Done.

**Comment by jbalboni** [comment]
@department-of-veterans-affairs/backend-review-group This needs another look.

File: config/settings.yml [hunk 1]
```diff
@@ -405,15 +405,19 @@ bing:
 
 flipper:
   admin_user_emails:
+    - alastair@adhocteam.us
     - anna@adhoc.team
-    - johnny@oddball.io
-    - travis.hilton@oddball.io
-    - rian.fowler@adhoc.team
-    - rianfowler@outlook.com
     - jbalboni@gmail.com
```
Comment (johnpaulashenfelter) [inline config/settings.yml]: Seems like gmail addresses might not be ones we want to be an admin

File: config/settings.yml [hunk 2]
```diff
@@ -405,15 +405,19 @@ bing:
...
     - jeff@adhocteam.us
+    - johnny@oddball.io
     - kam@adhocteam.us
-    - alastair@adhocteam.us
+    - lauren.alexanderson@va.gov
+    - mark.greenburg@adhocteam.us
+    - narin@adhocteam.us
+    - rian.fowler@adhoc.team
+    - rianfowler@outlook.com
```
Comment (johnpaulashenfelter) [inline config/settings.yml]: Do we need both of these, including one generic domain (outlook.com) for Rian?

File: config/settings.yml [hunk 3]
```diff
@@ -405,15 +405,19 @@ bing:
...
+    - ryan.thurlwell@va.gov
+    - travis.hilton@oddball.io
     - n.ratana@gmail.com
```
Comment (johnpaulashenfelter) [inline config/settings.yml]: Ditto the gmail comment above -- seems like should be consultancy or va.gov accounts

**Pull request merged**